RELEASE-NOTES.txt

```diff
@@ -1,7 +1,9 @@
 22.9
 -----
+* [*] [internal] Fix multiple memory leaks after logging in and logging out. [#21047]
+* [*] Performance improvements in login and logouts flows. [#21047]
 * [**] Block editor: Move undo/redo buttons to the navigation bar. [#20930]
-* [*] Fixed an issue that caused the UI to be briefly unresponsive in certian case when opening the app. [#21065]
+* [*] Fixed an issue that caused the UI to be briefly unresponsive in certain case when opening the app. [#21065]
```

Comment:
This might be here by accident since https://github.com/wordpress-mobile/WordPress-iOS/pull/21065 is already merged to `trunk`.

Reply:
Yes, it's already merged. I just fixed a typo `certian` -> `certain`